discatpy/models/user.py

```diff
@@ -30,6 +30,26 @@ class UserPremiumTypes(int, Enum):
 
 
 class UserFlags(Flag):
+    if t.TYPE_CHECKING:
+        def __init__(
+            self,
+            staff: bool = ...,
```

Comment:
You get the point

Reply:
[`b9449ce` (#18)](https://github.com/discatpy-dev/library/pull/18/commits/b9449ce65dede1ffe8bd714334dfb47f61533a39)